### ISSUE TITLE
[DC]Joystick disconnect after start a race in V-Rally 2 Expert Edition with rumble 

### ISSUE THREAD

**Issue by rpf16rj** [darthcloud/BlueRetro#322]
Hi, in Dreamcast on V-Rally 2: Expert Edition aftert start a race, aftert more or less 10 seconds, the joystick disconnect and not reconnect again, only after power off and power on, then connect. 
I beleave whit because of rumble feature.
In my tests, disable the rumble on config of Blueretro, work fine.

**Comment by darthcloud** [comment]
Yes dreamcast rumble is not well supported. Better left off for now.

**Comment by darthcloud** [comment]
Closing, DC rumble will be improved via #5

**Comment by rpf16rj** [comment]
Hi Darth. I aready tested the new version but this issue continue, the joystick is disconnecting.

**Comment by darthcloud** [comment]
I didn't say it was fixed, just said another issue is already open tracking that problem
